app/Actions/Place/GetPlaceCollectionByFilters/GetPlaceCollectionByFiltersAction.php

```diff
@@ -28,6 +28,8 @@ public function execute(GetPlaceCollectionByFiltersRequest $request): GetPlaceCo
     {
         $categoryId = $request->getCategoryId();
         $location = $request->getLocation();
+        $page = $request->getPage() ? : 1;
```

Comment:
`?:` no extra page. 
We can add special const PAGE_DEFAULT = 1 into GetPlaceCollectionByFiltersRequest